sdk/src/androidTest/java/co/omise/android/ParcelableTest.kt

```diff
@@ -0,0 +1,96 @@
+package co.omise.android
+
+import android.os.Bundle
+import android.os.Parcel
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.omise.android.models.*
```

Comment:
Could you use single import instead of wildcard import?

Reply:
👍

Reply:
After importing all one by one, the IDE is auto correcting it to wild card, so I guess it's best to leave it that way.

Reply:
I think you change it from the setting.

Reply:
We can, but if we are import 8/14 files in a package, it does kind of make sense to have this rule to use wildcard import

Reply:
So, how many import that we start to use wildcard?